tempest/src/main/kotlin/app/cash/tempest/internal/DynamoDbQueryable.kt

```diff
@@ -128,4 +132,20 @@ internal class DynamoDbQueryable<K : Any, I : Any>(
     val offsetKey = keyCodec.toApp(offsetKeyAttributes)
     return Offset(offsetKey)
   }
+
+  companion object {
+    // TODO: fill in the rest of the reserved words
+    private val reservedWordsMapping = mapOf(
+        "token" to "#t",
+        "year" to "#y"
+    );
+
+    private val reservedWordsAliasMapping = reservedWordsMapping.entries
```

Comment:
I think you can do 
```
val reversed = map.entries.associateBy({ it.value }) { it.key }
```
or 
```
val reversed = map.entries.associate{(k,v)-> v to k}
```
https://stackoverflow.com/questions/45380280/how-to-reverse-a-map-in-kotlin

Reply:
A) That's dope
B) The filtering update removed the need for this